libindy-crypto/src/cl/issuer.rs

```diff
@@ -4,8 +4,8 @@ use errors::IndyCryptoError;
 use pair::*;
 use cl::constants::*;
 use cl::helpers::*;
-use utils::commitment::*;
-use utils::get_hash_as_int;
+use cl::commitment::get_pedersen_commitment;
+use cl::hash::get_hash_as_int;
```

Comment:
What was the reason for this change?
Commitments are agnostic to any signature scheme.

Reply:
@mikelodder7 I can't recall the exact reason right now, but it's most likely because these functions were used only by the cl module. I wasn't really moving code (such as here) for logical structuring per se, but in order to avoid unnecessary dependencies within the bls module. For example, so long as the OpenSSL dependency was pulled in when building WASM bindings for bls, the build would fail.

I can check if the build fails if I move these functions back into the utils module, and get back to you.